DirectProgramming/DPC++FPGA/Tutorials/Features/scheduler_target_fmax/src/scheduler_target_fmax.cpp

```diff
@@ -0,0 +1,123 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <CL/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+
+// dpc_common.hpp can be found in the dev-utilities include folder.
+// e.g., $ONEAPI_ROOT/dev-utilities/include/dpc_common.hpp
+#include "dpc_common.hpp"
+
+using namespace sycl;
+
+constexpr unsigned seed = 1313;
+
+// Forward declare the kernel names in the global scope.
+// This FPGA best practice reduces name mangling in the optimization reports.
+class NoSchedulerTargetFMAX;
+class SchedulerTargetFMAX;
+
+// Runs the Kernel
+void KernelRun(size_t size, const std::vector<char> &input_data,
+               std::vector<unsigned> &output_data) {
+
+#if defined(FPGA_EMULATOR)
+  ext::intel::fpga_emulator_selector device_selector;
+#else
+  ext::intel::fpga_selector device_selector;
+#endif
+
+  try {
+    // create the SYCL device queue
+    queue q(device_selector, dpc_common::exception_handler,
+            property::queue::enable_profiling{});
+
+    buffer input_buffer(input_data);
+    buffer output_buffer(output_data);
+
+    auto e_g = q.submit([&](handler &h) {
+      accessor input_a(input_buffer, h, read_only);
+      accessor output_a(output_buffer, h, write_only, no_init);
+
+      h.single_task<NoSchedulerTargetFMAX>([=
```

Comment:
Have you run clang format on these files? The line split seems quite odd here.

Reply:
Yes I have. I'll rerun it in the next commit to guarantee.